metrics/prometheus.go

```diff
@@ -423,6 +423,14 @@ func NewPrometheusCollector(i infoProvider, f ContainerLabelsFunc, includedMetri
 					return metricValues{{value: float64(s.Memory.WorkingSet), timestamp: s.Timestamp}}
 				},
 			},
+			{
+				name:      "container_memory_migrate",
```

Comment:
this is still under `if includedMetrics.Has(container.MemoryUsageMetrics)`.

I think since we're introducing new CPUSet category of metrics, makes sense to add a new [metric kind](https://github.com/google/cadvisor/blob/bc8b91889a41c6f8f7b12819824d197ae0d57b1f/container/factory.go#L45-L66) for CpuSet metrics instead of putting this under existing container memory usage metrics.

Reply:
Done